### PR TITLE
Sort of porting MRE rations from Yogstation.

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
Sort of porting Yogstation PR #4757 with added improvements and (very very mild, that stuff is long-lasting) bad food poisoning chances if the mre expired for too long.

## Why It's Good For The Game
More maintenance scavenger stuff that isn't pp pillows, pp enlargment pills or free pp sutosurgeon.

## Changelog
:cl: Ghommie (Credits to Kmc2000 for the original PR)
add: Porting in MRE boxes from Yogstation. But be careful, eating possibly expired MREs found in maintenance comes with an unrealistically large (actually small) chance of food poisoning. Otherwise just bail out and order actually safe-to-eat MREs from cargo for 2000 credits.
/:cl:

### CODE REVIEW

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -1983,6 +1983,18 @@
 	considered <b>\[REDACTED\]</b> and returned at your leisure. Note that objects the anomaly produces are specifically attuned exactly to the individual opening the anomaly; regardless \
 	of species, the individual will find the object edible and it will taste great according to their personal definitions, which vary significantly based on person and species.")
 
+
+/datum/supply_pack/organic/mre
```
Comment (Trilbyspaceclone) [inline code/modules/cargo/packs.dm]: This is in the wrong place

File: code/game/objects/items/storage/boxes.dm [hunk 2]
```diff
@@ -1126,3 +1126,101 @@
...
+	new /obj/item/reagent_containers/food/drinks/coffee/type2(src)
+	new /obj/item/tank/internals/emergency_oxygen(src)
+
+/obj/item/reagent_containers/food/drinks/coffee/type2
+	name = "\improper Coffee, instant (type 2)"
+	desc = "Coffee that's been blow dried into a granulated powder. This packet includes self heating water for your nutritional pleasure."
+	icon = 'icons/obj/food/containers.dmi'
+	icon_state = "condi_cornoil"
+
+/obj/item/reagent_containers/food/snacks/pizzaslice/pepperoni
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/storage/boxes.dm]: Please place this and the other in the correct pizza files

File: code/game/objects/items/storage/boxes.dm [hunk 3]
```diff
@@ -1126,3 +1126,101 @@
...
+	name = "\improper MRE pepperoni pizza slice"
+	desc = "A freeze dried, dehydrated slice of bread with tomato sauce, pepperoni and cheese."
+	icon_state = "meatpizzaslice"
+	filling_color = "#A52A2A"
+	tastes = list("cardboard" = 1, "tomato" = 1, "cheese" = 1, "pepperoni" = 2)
+	foodtype = GRAIN | VEGETABLES | DAIRY | MEAT
+
+/obj/effect/spawner/lootdrop/mre
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/storage/boxes.dm]: Dose not belong in Box storage, rather loot spawn files would be better

File: code/modules/cargo/packs.dm (hunk 1 above)
Comment (Ghommie) [reply code/modules/cargo/packs.dm]: Where should it belong then?

Comment (Trilbyspaceclone) [reply code/modules/cargo/packs.dm]: One up, its not A->Z
Basicly above the pizza crate do to M coming before P iirc

**Review by silicons — APPROVED** [review]

**Pull request merged**